jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -31,7 +36,13 @@ public ChipAssert is() {
         return new ChipAssert().set(this);
     }
 
-    @JDIAction("Get {name}'s text")
+    @Override
+    @JDIAction("Check that '{name}' is enabled")
+    public boolean isEnabled() {
+        return !hasClass("v-chip--disabled");
+    }
+
+    @JDIAction("Get '{name}' text")
     public String getText() {
         if (!find(TEXT1).getText().isEmpty()) {
             return find(TEXT1).getText();
```

Comment:
Are we sure that we need this if/else? As I see in both cases we have .v-chip__content and should get the text.

Reply:
It seems that it works with TEXT1 even without condition, thanks!